src/App.vue

```diff
@@ -1,14 +1,6 @@
 <template>
   <v-app id="app">
     <div id="main-column">
-      <SbcAuthenticationOptionsDialog
-        attach="#app"
-        :inAuth="false"
-        :showModal="getIncorporateLoginModalVisible"
-        :redirectUrl="nameRequestUrl"
-        @close="setIncorporateLoginModalVisible(false)"
-      />
```

Comment:
Per Janis, this is obsolete. We now redirect to BC Registry site to login (or create a new account).